app/views/users/index.html.erb

```diff
@@ -1 +1,5 @@
-<h1>Here is a list of users</h1>
+<ul class='user-list'>
+  <% @users.each do |user| %>
+    <li><%= render 'partials/user_card', user: user %></li>
+  <% end %>
```

Comment:
- [ ] Kindly make sure when clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png). I can suggest You use the `link_to` method as explained [here](https://www.rubyguides.com/2019/05/rails-link_to-method/).